backend/portal/models.py

```diff
@@ -37,15 +37,22 @@ class Poll(models.Model):
     )
 
     club_code = models.CharField(max_length=255, blank=True)
```

Comment:
Hmm, not entirely sure of this myself. What are people's thoughts on having `club_code` and `club_comment` here? Do you think this is general enough for future models or should we push this back down to the `Poll` and `Post` models.

Reply:
wait what future models were you thinking of? I was thinking Content is just the "superclass" to Post/Polls, so has all the shared stuff. I guess you're thinking of a different "meaning" to Content?

Reply:
Hm that is fair. Yea I think posts and polls are really all that is it. I was thinking maybe Ashley's Events could also inherit but thinking about it some more that wouldn't make sense. Okay down to merge haha